test/Conversion/TorchToTosa/basic.mlir

```diff
@@ -2519,3 +2519,117 @@ func.func @torch.aten.replication_pad2d$basic(%arg0: !torch.vtensor<[1,1,3,3],f3
   %1 = torch.aten.replication_pad2d %arg0, %0 : !torch.vtensor<[1,1,3,3],f32>, !torch.list<int> -> !torch.vtensor<[1,1,10,6],f32>
   return %1 : !torch.vtensor<[1,1,10,6],f32>
 }
+
+// -----
+
+// CHECK-LABEL:   func.func @torch.aten.outer$basic(
```

Comment:
Can you also add a LIT test for `tanh` approximation for `Gelu`?

Reply:
Ack